stable/drone/templates/NOTES.txt

```diff
@@ -1,4 +1,4 @@
-{{- if hasKey .Values.server.env "DRONE_PROVIDER" }}
+{{- if eq (include "drone.providerOK" .) "true" }}
```

Comment:
I had trouble using the boolean results from the `drone.providerOK` template function, but it compares as a string okay, so I had to do it this way.